monitoring/frontend.go

```diff
@@ -446,13 +446,12 @@ func Frontend() *Container {
 							PossibleSolutions: "none",
 						},
 						{
-							Name:              "90th_percentile_updatecheck_requests",
-							Description:       "90th percentile successful update-check requests (sourcegraph.com only)",
-							Query:             `histogram_quantile(0.9, sum by (method,le) (rate(src_updatecheck_client_duration_seconds_bucket[5m])))`,
+							Name:              "total_time_to_perform_update_checks",
+							Description:       "amount of time to perform update check",
```

Comment:
```suggestion
							Name:              "update_check_duration",
							Description:       "update check duration",
```

(a bit more in line with our other wording)